api/build.gradle

```diff
@@ -8,8 +8,7 @@ buildscript {
     ext {
         GOOGLE_TRUTH_VERSION = '1.1.3'
         GSON_VERSION = '2.9.0'
-        HIBERNATE_VERSION = '5.6.9.Final'
-        JACKSON_DATABIND_VERSION = '2.13.4'
```

Comment:
this is only useful when we need to specify a different `Jackson Databind` version than the usual `Jackson` version.  We don't need to do this currently.